interface/forms/clinical_notes/templates/report.html.twig

```diff
@@ -0,0 +1,41 @@
+{#
+The Report template of the clinical note form.
+
+@package OpenEMR
+@subpackage Forms
+@author Robert Down <robertdown@live.com
+@copyright Copyright (c) 2023 Robert Down <robertdown@live.com>
+@copyright Copyright (c) 2023 Providence Healthtech
+@license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+#}
+<div class="w-100">
+    {% for n in notes %}
+        <div class="d-flex justify-content-between bg-light border border-dark p-1">
+            <div class="form-group mb-0">
+                <div class="font-weight-bold">{{ "Date"|xlt }}</div>
+                <div>{{ n.date|date() }}</div>
+            </div>
+            <div class="form-group mb-0">
+                <div class="font-weight-bold">{{ "Type"|xlt }}</div>
+                <div>{{ n.codetext|default("Unspecified"|xl)|text }}</div>
+            </div>
+            <div class="form-group mb-0">
+                <div class="font-weight-bold">{{ "Category"|xlt }}</div>
+                <div>{{ n.clinical_notes_category|default("Unspecified"|xl)|text }}</div>
+            </div>
+            <div class="form-group mb-0">
+                <div class="font-weight-bold">{{ "Author"|xlt }}</div>
+                <div>{{ n.user|text }}</div>
+            </div>
+            {% if n.code %}
+            <div class="form-group mb-0">
+                <div class="font-weight-bold">{{ "Code"|xlt }}</div>
+                <div>{{ n.code|text }}</div>
+            </div>
+            {% endif %}
+        </div>
+        <div>
+            {{ n.description|text|nl2br }}
+        </div>
```

Comment:
Consider using other html elements such as headers, p and span to make the view code more descriptive. It helps with accessibility too 🙂

Reply:
Wrapping that in a p tag would result in the entire note being one paragraph - not an ideal situation. I agree with you in principal, but practically it won't work here, paragraphs would need to be managed upstream.

There is a balance between enough progress to move the codebase forward and 100% perfect semantics - at the moment I'm leaning a bit more on practical over perfect. It's a long held battle for me, because I have been one to let perfect get in the way of good for many years.

Reply:
I see your point. I was mostly refering to the divs that hold data inside the loop, but I guess that's on me for inserting the comment at this point of the template. I don't think it's a blocker, done and functional is better than perfect

Reply:
You nudged me a bit... swapping out the outermost `div` for an `article role="document"` and inner `divs` to `header` and `section`